webview/localization.py

```diff
@@ -12,6 +12,7 @@
     "cocoa.menu.showAll": u"Show All",
     "cocoa.menu.quit": u"Quit",
     "cocoa.menu.fullscreen": u"Enter Fullscreen",
+    "cocoa.alert": u"Alert",
```

Comment:
Yes. This is unnecessary if we make the above proposed change. I shall commit once you follow up. 👍